apps/browser-extension-wallet/src/providers/AnalyticsProvider/postHog/PostHogClient.ts

```diff
@@ -83,12 +86,19 @@ export class PostHogClient {
       : DEV_NETWORK_ID_TO_POSTHOG_TOKEN_MAP[chain.networkMagic];
   }
 
+  protected getProjectID(): number {
```

Comment:
Small suggestion for improving consistency:
```suggestion
  protected getProjectId(): number {
```